SomosMasApp/SomosMasApp/App/Shared/Modules/TabBar/ViewController/TabBarViewController.swift

```diff
@@ -43,9 +43,17 @@ class TabBarViewController: UITabBarController {
         secondTabNavigationController.tabBarItem = UITabBarItem(title: "Nosotros", image: UIImage(systemName: "person.3"), selectedImage: nil)
         
         
+        // "News" Tab
+        
+        let tercerViewController = NewsViewController(nibName: "NewsViewController", bundle: nil)
+
+        let tercerTabNavigationController = UINavigationController(rootViewController: tercerViewController)
+
+        tercerTabNavigationController.tabBarItem = UITabBarItem(title: "News", image: UIImage(systemName: "newspaper"), selectedImage: nil)
+        
```

Comment:
you have to make a decide or you put in in a Tab that was not asked but you can or you push it when you push view more that in my opinion is better